pkg/validatormanager/weight_update.go

```diff
@@ -57,6 +61,7 @@ func InitializeValidatorWeightChange(
 
 func InitValidatorWeightChange(
 	ctx context.Context,
+	logger logging.Logger,
```

Comment:
Seems that currently we have printFunc -> where we use ux.Logger.PrintToUser. Do we want logger or printFunc? @felipemadero

Reply:
can you remove the printFunc usage in this PR?